### DISCUSSION TITLE
Option to favourite/star packages on website

### DISCUSSION THREAD

**Discussion by krush11** [npm/feedback#601] [Website] [answered]
It would be a great feature if registered users could star or favourite packages. In this case, we will be having a list of smaller packages that might be helpful to us in future. Currently, I find that if I find some package to be useful, I have to write it down somewhere so that I don't have to search it from scratch when I need it. A favourite button would be really great for this!!!!

**Comment by MylesBorins** [comment, marked as answer]
@krush11 starring / favoriting packages is not a feature we are actively investing in at the moment. If we decide in the future to expand the support we have for star / favorite we will definitely add a button on the website, but for now we will not be pursuing that work.

**Reply by markcellus** [reply]
Good to hear this isn't on the radar. IMHO these sorts of social-media-like features seem useless and just perpetuate popularity contests. If people need to save a package, just use bookmarks. They've been available and in existence like, forever now.
